core/services/workflows/engine.go

```diff
@@ -1191,22 +1191,23 @@ func (e *Engine) Name() string {
 }
 
 type Config struct {
-	Workflow             sdk.WorkflowSpec
-	WorkflowID           string
-	WorkflowOwner        string
-	WorkflowName         string
-	Lggr                 logger.Logger
-	Registry             core.CapabilitiesRegistry
-	MaxWorkerLimit       int
-	QueueSize            int
-	NewWorkerTimeout     time.Duration
-	MaxExecutionDuration time.Duration
-	Store                store.Store
-	Config               []byte
-	Binary               []byte
-	SecretsFetcher       secretsFetcher
-	HeartbeatCadence     time.Duration
-	StepTimeout          time.Duration
+	Workflow              sdk.WorkflowSpec
+	WorkflowID            string
+	WorkflowOwner         string
+	WorkflowName          string
```

Comment:
@justinkaseman Please add comments here detailing what should be used when:
- WorkflowName is the full, human-readable workflow name
- WorkflowNameTransform is responsible for translating the human-readable workflow name to its onchain format (i.e. hex encoded, max 10 bytes)